src/helpers/functions.ts

```diff
@@ -0,0 +1,2 @@
+export function noop() {
+}
```

Comment:
Should something be here?

Reply:
Nope.  This is intended to be a no-op function that can be used anywhere that a void function is needed without having to use `() => {}`.